enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx

```diff
@@ -46,6 +46,8 @@ const SdkContentWrapperInner = styled.div<
   --mb-color-shadow: ${({ theme }) => theme.fn.themeColor("shadow")};
   --mb-color-border: ${({ theme }) => theme.fn.themeColor("border")};
   --mb-color-text-dark: ${({ theme }) => theme.fn.themeColor("text-dark")};
+  --mb-color-text-medium: ${({ theme }) => theme.fn.themeColor("text-medium")};
+  --mb-color-text-light: ${({ theme }) => theme.fn.themeColor("text-light")};
```

Comment:
that means we can use `var(--mb-color-text-light)` directly instead of the `theme.fn.themeColor` in our children components, nice!